server/build.gradle.kts

```diff
@@ -91,6 +92,7 @@ tasks.register<JavaExec>("debugRun") {
     standardInput = System.`in`
 
     jvmArgs(debugArgs)
+    args(listOf("--tcpServerPort", serverDebugPort, "--tcpDebug", "--tracingLog"))
```

Comment:
Hm this changes the default from stdio, right? Does

```
./gradlew debugRun --args '--tcpServerPort ... --tcpDebug --tracingLog'
```

work? If so, I wouldn't change this.

Reply:
Seems not work..